src/vs/workbench/contrib/comments/browser/commentsView.ts

```diff
@@ -68,6 +70,58 @@ export class CommentsPanel extends FilterViewPane implements ICommentsView {
 
 	readonly onDidChangeVisibility = this.onDidChangeBodyVisibility;
 
+	get focusedCommentNode(): CommentNode | undefined {
+		const focused = this.tree?.getFocus();
+		if (focused?.length === 1 && focused[0] instanceof CommentNode) {
+			return focused[0];
+		}
+		return undefined;
+	}
+
+	get focusedCommentInfo(): string | undefined {
+		const focused = this.focusedCommentNode;
+		if (!focused) {
+			return;
+		}
+		return this.getScreenReaderInfoForNode(focused, 'accessibleViewContent');
```

Comment:
Can just check `this.focusedCommentNode` instead of assigning to `const focused` since nothing in this method is async.